README.md

```diff
@@ -1,31 +1,158 @@
 
-Melissa
+Melissa project
 ======
 
 Melissa offers optimization in the passage of messages between several limited resource nodes and a high-performance node for intensive processing (for example contour detection).
 
 To do this, you will use the ZeroMQ libraries together with Klepsydra, and the results can be seen with the OpenMCT display framework.
 
-- Since 2019-11-10
 
-melissa package
---------------
+# Installation instructions
+
+This section contains installation instructions
+
+    sudo apt --assume-yes install make
+    sudo apt --assume-yes install cmake
+    sudo apt --assume-yes install build-essential
+    sudo apt --assume-yes install dstat
+    sudo apt update
+
+    mkdir -p ~/thirdparties
+    cd ~/thirdparties/
 
-It is the main package, please see its [README][mRD].
+    git clone https://github.com/jbeder/yaml-cpp
+    cd yaml-cpp/
+    mkdir build
+    cd build
+    cmake -DYAML_BUILD_SHARED_LIBS=ON ..
+    make
+    sudo make install
 
-Here you may read [installation instructions][inst].
+# Install ZeroMQ
 
+Install ZeroMQ core engine in C++
+-----------------------------------
+    cd ~/thirdparties/
+    git clone https://github.com/zeromq/libzmq.git --branch latest_release
+    cd libzmq/
+    mkdir build
+    cd build
+    cmake ..
+    make
+    make test
+    sudo make install
 
-utils package
+Install ZeroMQ for C++
+-------------------------
+    cd ~/thirdparties/
+    git clone https://github.com/zeromq/cppzmq.git
+    cd cppzmq/
+    git checkout v4.5.0
+    mkdir build
+    cd build
+    cmake ..
+    make
+    make test
+    sudo make install
+
+
+Install ZeroMQ for C
 ----------------------
+    cd ~/thirdparties/
+    git clone git://github.com/zeromq/czmq.git
+    cd czmq
+    mkdir build
+    cd build
+    cmake ..
+    sudo make install
+
+Install googletest
+------------------
+    cd ~/thirdparties/
+    wget https://github.com/google/googletest/archive/release-1.10.0.zip
+    unzip release-1.10.0.zip
+    rm release-1.10.0.zip
+    mv googletest-release-1.10.0/ googletest
+
+Install openCV
+--------------
+    wget https://github.com/opencv/opencv/archive/3.4.1.tar.gz
+    tar zxvf 3.4.1.tar.gz
+    cd opencv-3.4.1
+    mkdir build
+    cd build
+    cmake -D CMAKE_BUILD_TYPE=RELEASE ..
+    make -j4
+    sudo make install
 
-This package contains some extra nodes and plugins for mavros, please see its [README][uRD].
+# Install klepsydra
 
-installation_instructions
+    rm -rf ~/thirdparties/klepsydra2
```

Comment:
no need to call it ```klepsydra2```. We could state in the readme that the folder could be called ```klepsydra```